components/properties/src/script.rs

```diff
@@ -317,12 +366,12 @@ impl<'data> ScriptWithExtensions<'data> {
     ///
     /// If `code_point` has Script_Extensions, then return the Script codes in
     /// the Script_Extensions. In this case, the Script property value
-    /// (normally Common or Inherited) is not included in the [`ZeroSlice`].
+    /// (normally Common or Inherited) is not included in the ScriptExtensionsSet.
```

Comment:
Nit: Put the name in brackets in order to retain the link and keep it up-to-date:

```
[`ScriptExtensionSet`]
```